examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -12813,111 +12813,111 @@
           "enabled": 0,
           "commands": [
             {
-              "name": "LockDoor",
+              "name": "Lock Door",
               "code": 0,
               "mfgCode": null,
               "source": "client",
               "incoming": 1,
               "outgoing": 1
             },
             {
-              "name": "UnlockDoor",
+              "name": "Unlock Door",
               "code": 1,
               "mfgCode": null,
               "source": "client",
               "incoming": 1,
               "outgoing": 0
             },
             {
-              "name": "UnlockWithTimeout",
+              "name": "Unlock with Timeout",
```

Comment:
i would favor to update the specs.... really

Reply:
+1

Reply:
I considered that, but it's a bit of a pain to do that in practice, compared with just changing the SDK.  I asked about this in the door lock channel; we'll see how that goes.

Reply:
OK, I filed https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/6190

If someone has time to fix up the spec, I would really appreciate it.

Reply:
yep hopefully someone step forward from this group, thanks Boris for the issue filled